kong.conf.default

```diff
@@ -1357,57 +1357,98 @@
+                                 #              them. The sandboxed function has
+                                 #              restricted access to the global
+                                 #              environment and only has access
+                                 #              to standard Lua functions that
+                                 #              will generally not cause harm to
+                                 #              the Kong Gateway node.
+                                 #
+                                 # * `on`: Functions have unrestricted
+                                 #         access to the global environment and
+                                 #         can load any Lua modules. This is
+                                 #         similar to the behavior in
+                                 #         Kong Gateway prior to 2.3.0.
+                                 #
+                                 # The default `sandbox` environment does not
+                                 # allow importing other modules or libraries,
+                                 # or executing anything at the OS level (for
+                                 # example, file read/write). The global
+                                 # environment is also not accessible.
+                                 #
+                                 # Examples of `untrusted_lua = sandbox`
+                                 # behavior:
+                                 #
+                                 # * You can't change global values such as
+                                 # `kong.license`.
+                                 # * You can run a local function:
+                                 # `local foo = 1 + 1`. However, running the
+                                 # same function at the OS level is not allowed:
+                                 # `os.execute('rm -rf /*')`.
+                                 #
+                                 # For a full allowed/disallowed list, see:
+                                 # https://github.com/kikito/sandbox.lua/blob/master/sandbox.lua
+                                 #
+                                 # To customize the sandbox environment, use
+                                 # the `untrusted_lua_sandbox_requires` and
+                                 # `untrusted_lua_sandbox_environment`
+                                 # parameters below.
+
+#untrusted_lua_sandbox_requires = # Comma-separated list of modules allowed to
+                                  # be loaded with `require` inside the
+                                  # sandboxed environment. Ignored
+                                  # if `untrusted_lua` is not `sandbox`.
+                                  #
+                                  # For example, say you have configured the
+                                  # Exit Transformer plugin and it contains the
```

Comment:
My bad when I used these examples. Exit transformer is not relevant to kong community edition.